ohapi/utils_fs.py

```diff
@@ -168,6 +168,82 @@ def load_metadata_csv(input_filepath):
     return metadata
 
 
+def is_valid_datetime(date):
+    """
+    Check if creation date is in ISO1069 format.
+
+    :param date: This field is creation date of the file.
+    """
+    regex = (r'^(-?(?:[1-9][0-9]*)?[0-9]{4})-(1[0-2]|0[1-9])-' +
```

Comment:
Since `arrow` is already used by this package, I think it should be used here, rather than create your own datetime validation. This would also make it consistent with the data validation happening on the main site; I dug up the line of code where this is checked in the API, and it's here: https://github.com/OpenHumans/open-humans/blob/master/private_sharing/forms.py#L261

(As a caveat, I think there would be non-ISO8601 strings may be valid with `arrow`, but nearly all ISO 8601 will be. Thus, the Open Humans API behavior itself is probably not completely consistent with its stated specifications, but hopefully close enough.)